src/bootstrap/test.rs

```diff
@@ -628,6 +628,10 @@ impl Step for Miri {
             cargo.env("MIRI_HOST_SYSROOT", sysroot);
             cargo.env("RUSTC_LIB_PATH", builder.rustc_libdir(compiler));
             cargo.env("MIRI", miri);
+            // propagate --bless
+            if builder.config.cmd.bless() {
+                cargo.env("MIRI_BLESS", "Gesundheit");
+            }
```

Comment:
This seems unrelated?

Reply:
Yeah it is. It makes Miri's test suite bless work with `x.py --bless`. Didn't seem significant enough for its own PR, but I can split it if you prefer.